### PR TITLE
Starter branch edge to edge implementation

### PR DESCRIPTION
**Pull request by the-scrambler**
Merge pending codelab update.

### CODE REVIEW

**Review by android-dev-lxl — APPROVED** [review]

File: app/src/main/java/com/example/dessertrelease/ui/DessertReleaseScreen.kt [hunk 1]
```diff
@@ -67,6 +68,39 @@ fun DessertReleaseApp(
     )
 }
 
+@Composable
```
Comment (johnshea) [inline app/src/main/java/com/example/dessertrelease/ui/DessertReleaseScreen.kt:71]: Unless there is a reason for moving the function here, move the function back to its original location in the file so the diff is smaller and only for the changes in the function body.

**Review by johnshea — CHANGES_REQUESTED** [review]
Navigation bar is not translucent on API 33 but it should be.

See [PR for branch `main-e2e`](https://github.com/google-developer-training/basic-android-kotlin-compose-training-dessert-release/pull/12) for screenshot and additional changes needed.

**Review by johnshea — APPROVED** [review]
LGTM but upgrade gradle wrapper to `8.2`

**Comment by android-dev-lxl** [comment]
Gradle update was needed

**Review by johnshea — APPROVED** [review]
lgtm

**Pull request merged**